bigbluebutton-html5/imports/ui/components/modals/dropdown/SettingsDropdown.jsx

```diff
@@ -0,0 +1,228 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import Modal from 'react-modal';
+import Icon from '/imports/ui/components/icon/component';
+import Button from '/imports/ui/components/button/component';
+import classNames from 'classnames';
+import styles from './styles';
+import SettingsModal from '../settings/SettingsModal';
+import SessionMenu from '../settings/submenus/SessionMenu';
+
+const customStyles = {
+  overlay: {
+    backgroundColor: 'rgba(0, 0, 0, 0)',
+  },
+  content: {
+    position: 'absolute',
+    top: '5%',
+    left: '83%',
+    right: '10%',
+    bottom: 'auto',
+    background: '#ffffff',
+    border: '1px solid #ffffff',
+    transform: 'translate(5%, 5%)',
+  },
+};
+
+export default class SettingsDropdown extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { isMenuOpen: false, };
+    this.menus = [];
+    this.openMenu = this.openMenu.bind(this);
+    this.closeMenu = this.closeMenu.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({ activeMenu: -1, focusMenu: 0, });
+    this.menus.push({ className: '',
+      props: { title: 'Fullscreen', prependIconName: 'icon-',
+                icon: 'bbb-full-screen', }, tabIndex: 1, });
+    this.menus.push({ className: SettingsModal,
+      props: { title: 'Settings/Options', prependIconName: 'icon-',
+                icon: 'bbb-more', }, tabIndex: 2, });
+    this.menus.push({ className: SessionMenu,
+      props: { title: 'Leave Session', prependIconName: 'icon-',
+                icon: 'bbb-logout', }, tabIndex: 3, });
+  }
+
+  componentDidMount() {
+    ReactDOM.render(
```

Comment:
Instead of `ReactDOM.render` It would be better if you imported this component and used it here: https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx#L54